lib/reporter.js

```diff
@@ -84,7 +84,8 @@ class CucumberReporter {
         case Cucumber.Status.PENDING:
         case Cucumber.Status.SKIPPED:
         case Cucumber.Status.AMBIGUOUS:
-            e = 'pending'
+            e = 'skipped'
```

Comment:
> The pending status is still there for really pending tests (tests with empty implementation)

According to this change, pending, skipped and ambiguous tests are still treated equally.